docs/kb/developer-resources/support-bundle-commands.md

```diff
@@ -0,0 +1,43 @@
++++
+date = "2016-08-12T00:00:00Z"
+lastmod = "2016-08-12T00:00:00Z"
+title = "Custom Support Bundle Commands Example"
+weight = "999999"
+categories = [ "Knowledgebase", "Developer Resources" ]
++++
+
+On-Prem Support Bundle allows you to leverage inline shell scripts in order to add valuable information into a support bundle. 
+*Note: Every individual command will have 30 seconds to execute and then subsequently timeout.*
+
+{{< warning title="Warning" >}}
```

Comment:
This isn't a relevant warning here.  Support commands run inside the application container, not on the host.